### PR TITLE
Enable use Ruby 2.6 with rack-dev-mark

### PR DESCRIPTION
**Pull request by mpg-shinichi-maeshima**
Ruby 2.6.0 released 🎉

### CODE REVIEW

File: rack-dev-mark.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |gem|
   gem.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   gem.require_paths = ['lib']
 
-  gem.required_ruby_version = ['>= 2.2', '< 2.6']
+  gem.required_ruby_version = ['>= 2.2', '< 2.7']
```
Comment (JunichiIto) [inline rack-dev-mark.gemspec]: Thank you for the update. But I am wondering why the upper limit is required. It's a little conservative and needs yearly update. So, how about changing like this?

``` ruby
gem.required_ruby_version = '>= 2.2'
# or
gem.required_ruby_version = ['>= 2.2', '< 3.0']
```

Comment (willnet) [reply rack-dev-mark.gemspec]: You have a point!

Comment (willnet) [reply rack-dev-mark.gemspec]: Removed upper limit 😃

**Comment by dtaniwaki** [comment]
Thank you for the fix. Although the CI failed, it is because new version of bundler dropped Ruby 2.2 support. I will drop the support in this gem as well in another PR.

**Pull request merged**